_docs/02.4-build_component.markdown

```diff
@@ -7,11 +7,14 @@ toplevel: "Getting Started: Quick Guide"
 
 After you've generated your awesome electrode app, you are ready to focus on writing your React components.
 
-Navigate to `<your-awesome-app>/client/components/home.jsx`:
+Navigate to `<your-awesome-app>/src/client/components/home.jsx`:
 
 ```javascript
 import React, {PropTypes} from "react";
 import {connect} from "react-redux";
+/**/
+import Notifications from "react-notify-toast";
```

Comment:
Something needs to change, as the docs don't match the generated code...  Should I change the generated code then to remove Notifications instead?